packages/scandipwa/src/component/ProductDownloadableLinks/ProductDownloadableLinks.component.js

```diff
@@ -30,43 +33,76 @@ export class ProductDownloadableLinks extends PureComponent {
 
     getLabel(link) {
         const { title, price } = link;
+        const { isRequired } = this.props;
+
+        if (!isRequired) {
+            return title;
+        }
 
-        return `${ title } (${ formatPrice(price) })`;
+        return `${ title } (+${ formatPrice(price) })`;
     }
 
-    renderCheckBox(link) {
-        const { sample_url, id } = link;
-        const { setSelectedCheckboxValues } = this.props;
+    renderLabel(link) {
+        const { sample_url } = link;
+
+        if (!sample_url) {
+            return this.getLabel(link);
+        }
 
         return (
-            <div>
-                <Field
-                  type="checkbox"
-                  label={ this.getLabel(link) }
-                  key={ id }
-                  id={ `link-${id}` }
-                  name={ `link-${id}` }
-                  value={ id }
-                  onChange={ setSelectedCheckboxValues }
-                />
-                <Link to={ sample_url } block="ProductDownloadableLinks" elem="SampleLink">
+            <>
+                { this.getLabel(link) }
+                <Link to={ sample_url } block="ProductDownloadableLink" elem="SampleLink">
                     { __('Sample') }
                 </Link>
+            </>
+        );
+    }
+
+    renderCheckBox(link) {
+        const { setSelectedCheckboxValues, isRequired } = this.props;
+        const { id } = link;
+
+        if (!isRequired) {
+            return null;
+        }
+
+        return (
+            <Field
+              type="checkbox"
+              key={ id }
+              id={ `link-${ id }` }
+              name={ `link-${ id }` }
+              value={ id }
+              onChange={ setSelectedCheckboxValues }
+            />
+        );
+    }
+
+    renderLink(link) {
+        return (
+            <div block="ProductDownloadableLink">
+                { this.renderCheckBox(link) }
+                <span block="ProductDownloadableLink" elem="SampleLabel">
+                    { this.renderLabel(link) }
+                </span>
             </div>
         );
     }
 
-    renderCheckBoxes() {
+    renderLinks() {
         const { links } = this.props;
 
-        return links.map((link) => this.renderCheckBox(link));
+        return links.map((link) => this.renderLink(link));
```

Comment:
It is preferable not to use arrow functions in render. Prefer declaring this directly like so:
```
links.map(this.renderLink);
```

And make sure the `renderLink` is defined as an arrow function.